v1/linkml-schemas/lifeEvents.yaml

```diff
@@ -0,0 +1,99 @@
+id: https://vocab.account.gov.uk/linkml/life-events-schema
+name: lifeEvents
+version: 1.0
+description: >-
+  Vocabulary for life events
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  vc: https://www.w3.org/2018/credentials/v1/
+  di_vocab: https://vocab.account.gov.uk/v1/
+imports:
+  - linkml:types
+  - ./events
+  - ./person
+default_curi_maps:
+  - semweb_context
+default_prefix: di_vocab
+default_range: string
+
+classes:
+  DeathRegistrationJWTClass:
+    is_a: SETClass
+    attributes:
+      events:
+        range: DeathRegistrationEventMappingClass
+        required: true
+  DeathRegistrationEventMappingClass:
+    attributes:
+      "deathRegistrationUpdate":
```

Comment:
Is this right? I think this should be Event, not Update?

Reply:
Ah you're right, this mistake will never have really noticed because of the alias.

I think just `deathRegistration` as that's consistent with the alias, though it's arguable whether "event" should be on the end of both (if we change it, we'd change the update event too).